src/base/File.cc

```diff
@@ -34,15 +34,15 @@ FileOpeningConfig::ReadOnly()
     FileOpeningConfig cfg;
 
     /* I/O */
-#if _SQUID_WINDOWS_
+#if _SQUID_WINDOWS_ || _SQUID_MINGW_
```

Comment:
This condition should not be added because its addition changes nothing in the official code: In the official code, `_SQUID_MINGW_` cannot be true if `_SQUID_WINDOWS_` is false. The situation will change after PR #1341 lands. AFAICT, this PR only changes these conditions, so, if they are removed, we will be left with the official code. Thus, there is nothing to merge here until PR 1341 lands.